app/src/components/dashboard/leaderboard.tsx

```diff
@@ -56,6 +56,17 @@ function sortRows(rows: Row[], variant: Variant, mode: Mode): Row[] {
     return copy;
 }
 
+function getYouTubeUrl(handle: string): string {
+    // Remove @ symbol if present at the beginning
+    const cleanHandle = handle.startsWith('@') ? handle.slice(1) : handle;
+    return `https://www.youtube.com/@${cleanHandle}`;
+}
+
+function openYouTubeChannel(handle: string): void {
```

Comment:
can we instead wrap the element in an `<a>` to avoid having to use an onclick and javascript navigation? i think that will play better with accessibiity readers, etc...